src/Components/Navbar/Navbar.css

```diff
@@ -92,14 +86,34 @@ h1 {
   height: 100%;
 }
 
-.title {
+.title-container {
   left: calc(50% - 13vh);
   position: absolute;
   font-family: 'Eczar', serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 11vh;
+}
+
+.title {
+  margin-left: 0;
+  margin-bottom: 0;
+  margin-top: 2%;
+  border-bottom: 1px solid black;
+  font-size: 3rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  color: rgb(70, 112, 130);
+}
+
+.subtitle {
+  margin-top: 0;
```

Comment:
1rem instead of 0
Needs to be responsive. Change font size for when screen gets smaller. Crushing it!